tests/dmctl_tools/dmctl_operate_task.go

```diff
@@ -15,18 +15,28 @@ package main
 
 import (
 	"context"
+	"fmt"
 	"os"
+	"strconv"
 
+	"github.com/pingcap/dm/dm/pb"
 	"github.com/pingcap/dm/tests/utils"
 )
 
 func main() {
+	if len(os.Args) != 3 {
+		utils.ExitWithError(fmt.Errorf("invlid args: %v", os.Args))
+	}
 	cli, err := utils.CreateDmCtl("127.0.0.1:8261")
 	if err != nil {
 		utils.ExitWithError(err)
 	}
-	conf := os.Args[1]
-	err = utils.StartTask(context.Background(), cli, conf, nil)
+	op, err := strconv.ParseInt(os.Args[1], 10, 64)
```

Comment:
can we use `string` as type of  argumen `op`? and then convert `op` to `pb.TaskOp` in this function

Reply:
changed to use string type argument, feel free to covert to pb.TaskOp then